### DISCUSSION TITLE
How do I represent a date like 5th of current month?

### DISCUSSION THREAD

**Discussion by ganesshkumar** [SilentVoid13/Templater#764] [Help]
I had gone through the documentation but couldn't find the exact syntax for creating a date like 5th or 10th of this month. Is an any example for this?

**Comment by liamcain** [comment]
You can just use moment for that:

```
<% window.moment({ day: 5 }).format('YYYY-MM-DD') %>
```

If you just want to display it, you don't even need Templater for this. You could use the core Templates plugin:

```
{{date:YYYY-MM-05}}
```
